systemd/osdf-cache.yaml

```diff
@@ -0,0 +1,28 @@
+## Enable Debug to send detailed logs, including XRootD logs, to the log file
+# Debug: false
+Logging:
+  LogLocation: /var/log/osdf-cache.log
+  ## Valid Levels are Trace, Debug, Info, Warning, Error, Fatal and Panic.
+  # Level: "Error"
```

Comment:
Not really related to this PR but do logs also go to the journal or just to the location here?

Reply:
Only to the log location.